### PR TITLE
LPS-171776 Create checks in Actions Return to Open

### PR DESCRIPTION
**Pull request by rafaelfs98**
[LPS-171488](https://issues.liferay.com/browse/LPS-171488)

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx [hunk 1]
```diff
@@ -65,11 +65,12 @@ const useTestflowActions = () => {
 						}
 					)
 				),
-			disabled: true,
+
```
Comment (rafaelfs98) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx:68]: Remove Space

Comment (rafaelfs98) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx:68]:
```suggestion
Remove space
```

**Review by rafaelfs98 — COMMENTED** [review]
:fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx [hunk 2]
```diff
@@ -65,11 +65,12 @@ const useTestflowActions = () => {
...
 			hidden: ({dueStatus}) =>
 				dueStatus?.key === TaskStatuses.IN_ANALYSIS,
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/useTestflowActions.tsx:70]: Could check the need for "?", if dueStatus is a required field.


```suggestion
				dueStatus.key === TaskStatuses.IN_ANALYSIS,
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestraySubtask.ts [hunk 3]
```diff
@@ -327,6 +328,7 @@ class TestraySubtaskImpl extends Rest<SubtaskForm, TestraySubTask> {
 		}
 
 		const updatedSubtask = await this.update(subTaskId, {
+			dueStatus: currentSubtask?.dueStatus.key,
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestraySubtask.ts:331]: Could check the need for "?", if dueStatus is a required field.


```suggestion
			dueStatus: currentSubtask.dueStatus.key,
```

Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/services/rest/TestraySubtask.ts:331]: Could you check the need for "?", in currentSubtask if dueStatus is a required field.


```suggestion
			dueStatus: currentSubtask.dueStatus.key,
```

**Review by samuelleolopes — COMMENTED** [review]
Good Job! :fire:

**Review by caiosouza15 — APPROVED** [review]
Great : fire:

**Review by welitonpe — APPROVED** [review]
Good Work  :fire::fire::fire:

**Comment by rafaelfs98** [comment]
@kevenleone  
I request your Review

**Comment by kevenleone** [comment]
ci:forward

**Comment by joelrocha68** [comment]
Just starting reviewing :)

**Pull request closed without merging**